### PR TITLE
Update notification payload for cve

### PR DESCRIPTION
**Pull request by GeetikaBatra**
Signed-off-by: Geetika Batra <gbatra@redhat.com

### CODE REVIEW

**Comment by GeetikaBatra** [comment]
[test]

File: src/notification/user_notification.py [hunk 1]
```diff
@@ -40,10 +40,20 @@ def generate_notification(report):
...
+            if is_transitive:
+                transitive_updates.append(deps)
+            else:
+                direct_updates.append(deps)
             total_cve_count += int(deps['cve_count'])
         result["data"]["attributes"]["custom"]["cve_count"] = total_cve_count
-
+        result["data"]["attributes"]["custom"]["transitive_updates"] = transitive_updates
+        result["data"]["attributes"]["custom"]["direct_updates"] = direct_updates
+        print("********************PAYLOAD****************************", result)
```
Comment (samuzzal-choudhury) [inline src/notification/user_notification.py]: @GeetikaBatra can this message be more meaningful? Also, I am assuming that `print` is being used temporarily and will be removed after a thorough testing of notification is done on staging, isn't it?

Comment (GeetikaBatra) [reply src/notification/user_notification.py]: @samuzzal-choudhury Yep this is temporary, I will be removing it as soon as I know that this piece of code works fine. Do I still need to make this message more meaningful?

Comment (samuzzal-choudhury) [reply src/notification/user_notification.py]: @GeetikaBatra yes, a log with something with more meaningful.

Comment (GeetikaBatra) [reply src/notification/user_notification.py]: @samuzzal-choudhury, does log statements appear in the terminal. I have observed that they are missed in staging logs, that's why I am using print statements.

Comment (samuzzal-choudhury) [reply src/notification/user_notification.py]: Better option will be to introduce FLASK_LOGGING_LEVEL env variable and replace the print with logger

File: openshift/template.yaml [hunk 2]
```diff
@@ -38,6 +38,8 @@ objects:
             value: bayesian-api
           - name: COREAPI_SERVICE_PORT
             value: "5000"
+          - name: FLASK_LOGGING_LEVEL
+            value: ${FLASK_LOGGING_LEVEL}
```
Comment (samuzzal-choudhury) [inline openshift/template.yaml]: @GeetikaBatra FLASK_LOGGING_LEVEL has to be defined with a value in the parameter section. [Here is an example](https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/openshift/template.yaml#L185-L189) of how to do that.

File: src/notification/__init__.py [hunk 3]
```diff
@@ -1 +1,4 @@
 """Handles user notification service."""
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```
Comment (samuzzal-choudhury) [inline src/notification/__init__.py]: @GeetikaBatra please look into linter, docstyle and build failures.

**Comment by GeetikaBatra** [comment]
@samuzzal-choudhury Thanks

**Pull request merged**